pydash/pydash_web/controller/dashboards.py

```diff
@@ -23,9 +23,9 @@ def dashboard(dashboard_id):
     try:
         db = pydash_app.dashboard.find(dashboard_id)
     except KeyError:
-        return jsonify({"message": "Could not find a matching dashboard."}), 404
-    except ValueError: # Happens when called without a proper UUID -- return test data for now in this case.
-        return jsonify(_json_mock_dashboard_detail())
+        return jsonify({"message": "Could not find a matching dashboard."}), 500
```

Comment:
Why is this an error 500? It's the client's fault, is it not? I think this should be a `404`.

Reply:
Fair point, I'll revert it back to its original status code.